unison-src/transcripts/fix5055.output.md

```diff
@@ -0,0 +1,93 @@
+```ucm
+.> builtins.merge
```

Comment:
I tried to leave this comment, but it seems to be lost in the void. It might make sense to create a small static namespace and `ls` it so that this transcript doesn't churn and create noise each time that a builtin is added.

Reply:
Fixed. How’s the new one?